core/src/com/unciv/ui/worldscreen/unit/UnitActions.kt

```diff
@@ -462,22 +462,24 @@ object UnitActions {
 
     private fun addSpreadReligionActions(unit: MapUnit, actionList: ArrayList<UnitAction>, tile: TileInfo) {
         if (!unit.hasUnique("Can spread religion [] times")) return
-        if (unit.religion == null) return
+        if (unit.religion == null || unit.civInfo.gameInfo.religions[unit.religion]!!.isPantheon()) return
         val maxReligionSpreads = unit.maxReligionSpreads()
         if (!unit.abilityUsedCount.containsKey("Religion Spread")) return // This should be impossible anyways, but just in case
         if (maxReligionSpreads <= unit.abilityUsedCount["Religion Spread"]!!) return
         val city = tile.getCity()
+        if (city == null) return
         actionList += UnitAction(UnitActionType.SpreadReligion,
             title = "Spread [${unit.religion!!}]",
             action = {
                 unit.abilityUsedCount["Religion Spread"] = unit.abilityUsedCount["Religion Spread"]!! + 1
-                city!!.religion[unit.religion!!] = 100
+                city.religion.clearAllPressures()
+                city.religion.addPressure(unit.religion!!, 100)
                 unit.currentMovement = 0f
                 if (unit.abilityUsedCount["Religion Spread"] == maxReligionSpreads) {
                     addGoldPerGreatPersonUsage(unit.civInfo)
                     unit.destroy()
                 }
-            }.takeIf { unit.currentMovement > 0 && city != null && city.civInfo == unit.civInfo } // For now you can only convert your own cities
```

Comment:
Awww.... OK, if this is interim contradicting the intended final rules I'd make that a //todo one, but that's just a matter of taste. We'll have to do a todo week sometime... 😆

Reply:
The line change here actually rectifies a previous interim ruling, so the new rules should be correct. I'll add a ToDo to other places where there is an interim ruling as of yet, though